src/conversion/downcast-converters.js

```diff
@@ -355,17 +358,19 @@ function _normalizeToElementConfig( config, ViewElementClass ) {
 		return;
 	}
 
+	const view = config.view;
+
 	// Build `.view` property.
-	// It is expected to be either creator function or view element instance.
-	if ( typeof config.view == 'string' ) {
-		// If `.view` is a string, create a proper view element instance out of given `ViewElementClass` and name given in `.view`.
-		config.view = new ViewElementClass( config.view );
-	} else if ( typeof config.view == 'object' && !( config.view instanceof ViewElementClass ) ) {
+	// It is expected to be either string, element definition or creator function.
+	if ( typeof view == 'string' ) {
+		// If `.view` is a string, create a function that returns view element instance out of given `ViewElementClass`.
+		config.view = () => new ViewElementClass( view );
```

Comment:
Shouldn't that be build using writer too?

Reply:
BTW., we also need to change `ViewElementClass` to some kind of flag or something that would then result in using a proper method from `view.Writer`.

Reply:
I was thinking about it but wasn't sure if going with a flag is a good option. Other than that this is private function used in engine and constructors will be protected to that package and we know what we are doing here. Buuut on the other hand (when I am thinking about it now after you mentioned it) changing it to writer will show that this is the proper practice everywhere :D I will take care of it.

Reply:
I will wait with this in follow up, after we unify callbacks for `downcastElementToElement` and `downcastAttributeToElement` - it will be easier to use `view.Writer` then.